toolkit/imageconfigs/marketplace-gen2.json

```diff
@@ -2,7 +2,7 @@
     "Disks": [
         {
             "PartitionTableType": "gpt",
-            "MaxSize": 1500,
+            "MaxSize": 5000,
```

Comment:
Is that the only image we want to change? What about the gen1 and ARM64 variants?

Reply:
I was told by Chris that this is only needed for the marketplace image that supports ADE extension work.